tests/integration/test_col_functions.py

```diff
@@ -526,70 +760,81 @@ def test_col_is_not_null_and_not_empty_array(spark):
         is_not_null_and_not_empty_array("timestamp_col"),
         is_not_null_and_not_empty_array("date_col"),
         is_not_null_and_not_empty_array("struct_col"),
+        is_not_null_and_not_empty_array(F.col("nested_array_col").getItem("arr")),
```

Comment:
any reason we have the test for the map as a separate test `test_col_is_not_null_map_arguments`?